UPGRADING.md

```diff
@@ -1,5 +1,9 @@
 # Upgrading Alexa-app
 
+### Upgrading to >= 5.0.0
+
+* Drop support for all NodeJS versions lower than 10
```

Comment:
You should make this a complete sentence. 

_Node.js 10 or later is now required._